static/openapi/prerendering-spec.json

```diff
@@ -1989,11 +1987,195 @@
         }
       }
     },
-    "/api/prerender/v1/origins/{_key}": {
+    "/api/prerender/v1/origins/{origin}": {
+      "get": {
+        "summary": "Get origin",
+        "tags": ["Origins"],
+        "description": "This endpoint returns an origin from the origin-related GDN fabric.",
+        "parameters": [
+          {
+            "schema": { "type": "string", "minLength": 1 },
+            "in": "path",
+            "name": "origin",
+            "required": true,
+            "description": "Origin hostname"
+          },
+          {
+            "schema": { "type": "string" },
+            "in": "header",
+            "name": "x-photoniq-customerid",
+            "required": true,
+            "description": "Customer unique identifier"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "description": "Successful response",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "Success status"
+                    },
+                    "message": {
+                      "type": "string",
+                      "description": "Success message"
+                    },
+                    "data": {
+                      "type": "object",
+                      "properties": {
+                        "_key": {
+                          "type": "string",
+                          "description": "Origin unique identifier"
+                        },
+                        "origin": {
+                          "type": "string",
+                          "format": "uri",
+                          "description": "Origin base URL"
+                        },
+                        "siteMaps": {
+                          "type": "array",
+                          "items": {
+                            "type": "string",
+                            "description": "Sitemap path"
+                          },
+                          "description": "List of sitemap paths"
+                        },
+                        "identifyNotfound": {
+                          "type": "string",
+                          "description": "Content that identifies not found pages (404) in rendered pages"
+                        },
+                        "allowOriginHeaderValue": {
+                          "type": "string",
+                          "description": "Value of X-PhotonIQ-Prerender-Request header"
```

Comment:
Should this header name of capitals?

Reply:
Headers are generally case-insensitive in web servers, but I frequently see them capitalized in other documentation. Let me know if you want to change it. In C8 documentation we have `X-C8-Error-Codes`, for example.

Reply:
I would say we now standardize headers are always lowercase.

Reply:
Done